pages/list.js

```diff
@@ -0,0 +1,12 @@
+import Link from "next/link";
+import TodoForm from "../components/ToDoList";
+
+export default function FormList({ listItems, setListItems }) {
+  return (
+    <>
+      <h1>Aufgabe hinzufügen</h1>
+      <TodoForm listItems={listItems} setListItems={setListItems} />
+      <Link href="/">Liste</Link>
```

Comment:
you're routing back to the same page "/" after adding a new list item. This is unnecessary